styles.css

```diff
@@ -0,0 +1,176 @@
+:root {
+    --plonk-dark-green: #0c5424;
+    --plonk-light-green: #245739;
+    --plonk-dark-pink: #f09894;
+    --plonk-medium-pink: #fab7ae;
+    --plonk-light-pink: #fcd3ce;
+}
+
+body {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 10px;
+    padding: 20px;
+    background-color: #000000;
+}
+
+header {
+    display: flex;
+    justify-content: center;
```

Comment:
Personally I'd say flex is overkill here - you could achieve the same result by just setting `text-align: center`